### ISSUE TITLE
Gauge chart text.replace is not a function after 4.10.1

### ISSUE THREAD

**Issue by Ecksters** [amcharts/amcharts4#3020]
**Bug description**
![image](https://user-images.githubusercontent.com/2662835/96771623-2962ca00-139f-11eb-8133-dc71cb5dde06.png)

```
TypeError: text.replace is not a function
    at Module.plainText (Utils.js?2082:482)
    at AxisLabelCircular.get [as readerTitle] (Label.js?028d:1378)
    at AxisLabelCircular.Sprite.applyAccessibility (Sprite.js?5d2f:3505)
    at AxisLabelCircular.set [as parent] (Sprite.js?5d2f:1065)
    at ValueAxis.Axis.appendDataItem (Axis.js?e68a:771)
    at eval (Axis.js?e68a:813)
    at eval (Iterator.js?0357:57)
    at eval (Iterator.js?0357:10)
    at Module.each (Iterator.js?0357:56)
    at ValueAxis.Axis.validateAxisRanges (Axis.js?e68a:812)
    at ValueAxis.validate (ValueAxis.js?364d:339)
    at ValueAxis.Sprite._systemUpdate (Sprite.js?5d2f:8645)
    at ValueAxis.Container._systemUpdate (Container.js?be33:1673)
    at eval (System.js?72b5:242)
    at eval (Object.js?a0dd:107)
    at eachContinue (Object.js?a0dd:93)
```

Upgrade from v4.10.0 to v4.10.1 gives us this error on our Gauge charts. The error is also present on 4.10.2 through 4.10.9.

**Environment (if applicable)**

Chrome Version 86.0.4240.75 (Official Build) (x86_64)

**Comment by martynasma** [comment]
Would you be able to post your whole chart and data on CodePen/jsFiddle so that we can test?

**Comment by Ecksters** [comment]
Reproduced in 4.10.9 as well, trying to get a codepen that can do it

**Comment by Ecksters** [comment]
It happens when this function attempts to call `.replace` on what is apparently a non-string:
```
export function plainText(text) {
    return text ? stripTags(text.replace(/[\n\r]+/g, ". ")) : text;
}
```

Managed to reproduce it:
https://codesandbox.io/s/sample-amcharts4-queuing-forked-7j0yq?file=/src/MetricTile.js

Seems like it's tied to the data.sizeX and data.sizeY values being large enough, if I make them smaller I don't get the error.

**Comment by martynasma** [comment]
Thanks. We'll fix this in the next version.

**Comment by martynasma** [comment]
Fixed in [4.10.10](https://github.com/amcharts/amcharts4/releases/tag/4.10.10).

## [4.10.10] - 2020-11-03

### Changed
- License check for `MapChart` was not working. If you are using commercial `MapChart`, make sure you use `addLicense()` for both Charts and Maps products.

### Fixed
- Better dispose check for delayed `"out"` event.
- Applying accessibility to some labels with only numeric information was resulting in error.
- Chart could result in an SO in some rare cases.
- Using `strictMinMax` with `includeRangesInMinMax` sometimes could caulse wrong zooms.
- In some cases Legend's item labels were being truncated when cahrt container was shrinking, but were not being restored to their full width once it was enlarged back.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
